### PR TITLE
Accessibility feature

### PR DESCRIPTION
**Pull request by fauziyahzeeyah**
-  Add a popup window
-  Update styles
-  Add object to Javascript
-  Made some changes in HTML
-  Add alt to img

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,398 +9,165 @@
...
+      </ul>
+    </section>
+
+    <section class="modal hidden" id="modal">
+      <div class="modal__wrapper">
+        <div class="modal__close" id="close-modal"><img src="img/close-icon.png" alt="close-icon"></div>
+        <div class="modal__content" id="modal-content">
         </div>
       </div>
-     </div>
```
Comment (Tadesse-Alemayehu) [inline index.html:401]: #### Good job on displaying the modal @fauziyahzeeyah. I tried to open your project both locally and live I have seen some minor details needed to be fixed. 
- [ ] Your images are not displayed, which makes your modal a bit off compared to the Figma design.
- [ ] I believe your design could be more responsive, on a screen size of `800px` I see your footer text is overflowing. Please address this for a better-looking modal. 

### Your popup on a screen size of `800px`
Screenshot 2022-12-15 084416

**Review by Tadesse-Alemayehu — CHANGES_REQUESTED** [review]
### Hi @fauziyahzeeyah 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-done-ashley-garcia


## Highlights
 - [ ] Good job overall following the template design 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  Good commit history 🥇
 - [ ]  Good PR title/description 💯
 - [ ]  All linters passed 🥇
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by fauziyahzeeyah** [comment]
-  Add images 
-  Removed auto overflow from text

**Review by emmiiorji — APPROVED** [review]
Hi @fauziyahzeeyah, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Congrats

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow
- Requested changes have been implemented


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**